packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -707,6 +707,44 @@ export const guides = [
           },
         ],
       },
+      {
+        title: "Automate managing with Terraform",
+        guideName: "terraform-landing-page",
+        subLinks: [
+          {
+            title: "Terraform overview",
+            guideName: "terraform-overview",
+          },
+          {
+            title: "Enable Terraform access",
+            guideName: "terraform-enable-org-access",
+          },
+          {
+            title: "Manage user access",
+            guideName: "terraform-manage-user-access",
+          },
+          {
+            title: "Manage groups",
+            guideName: "terraform-manage-groups",
+          },
+          {
+            title: "Manage authentication services",
+            guideName: "terraform-manage-external-authenticators",
+          },
+          {
+            title: "Manager end-user experience",
```

Comment:
```suggestion
            title: "Manage end-user experience",
```

Reply:
Thanks... and I caught that one while fixing another bug from Brent. It's just not committed yet.